src/sentry/static/sentry/app/components/events/interfaces/breadcrumbs.jsx

```diff
@@ -68,6 +68,7 @@ class BreadcrumbsInterface extends React.Component {
     // reverse array to get consistent idx between collapsed/expanded state
     // (indexes begin and increment from last breadcrumb)
     return crumbs
+      .sort((a, b) => a.timestamp > b.timestamp)
```

Comment:
The results in Percy suggest this isn't sorting correctly.

`.sort` should return -1, 0, or 1. Is it possible this is the cause?